### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by ramiaquista**
## What I have done: 🔨
- Created tasks-handler.js file for new methods.
- Added addTask() function to add new tasks.
- Added updateDescription() function to update the content.
- Added removeTask() function to remove a task.
- Added function to clear all completed tasks.
- Added localStorage functions to handle local data.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,60 +2,80 @@
...
+        icon.classList.add('hidden');
+      } else {
+        li.classList.remove('task-pressed');
+        icon.classList.remove('hidden');
+        trashIcon.classList.add('hidden');
+      }
+    });
+
+    labelDes.addEventListener('input', () => {
+      Crud.updateDescription(toDoTasks, li.id, labelDes.innerHTML);
```
Comment (henatan99) [inline src/index.js:70]: - [ ] I don't think an `innerHTML` can be passed as an input argument here. When the description is edited it assigns not just the string the user inputs but also the HTML elements too. You can see this effect on the attached screenshot on the localStorage. :pray: 

Screenshot from 2021-09-24 01-20-46

File: src/index.js [hunk 2]
```diff
@@ -85,6 +105,23 @@ function taskListDisplayed() {
...
+    if (task.completed) {
+      completedTasks.push(task);
+    }
+  });
+  completedTasks.forEach((task) => {
+    Crud.removeTask(toDoTasks, task.index);
+  });
+  setLocalStorage();
+  taskListDisplayed();
+}
```
Comment (henatan99) [inline src/index.js:123]: - [ ] You should implement a function for the "Clear all completed" button using filter() method as it is the project's requirement. It is more efficient. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @ramiaquista , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (ramiaquista) [reply src/index.js]: **Thank's @henatan99  for the review, I have solved it! 😄**

**Review by barackm — APPROVED** [review]
Hi @ramiaquista, 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**